### DISCUSSION TITLE
Correction to Github Rest Api V3 documentation `curl -i`

### DISCUSSION THREAD

**Discussion by AriESQ** [community/community#24537] [Apps, API and Webhooks]
<p>This is maybe a small gripe, but it is correct. The Github <a href="https://developer.github.com/v3/" rel="nofollow noopener">REST API v3 documentation</a> gives several examples such as <code>curl -i https://api.github.com -u foo:bar</code></p>
<p>However the <code>curl</code> documentation does not document a <code>-i</code> (lowercase) flag. Curl <a href="https://curl.haxx.se/docs/manpage.html#-I" rel="nofollow noopener">documents</a> the flag as <code>-I</code> (uppercase).</p>
<p>I’ll note that  <code>curl</code> currently accepts the lowercase flag, so there is little impact to this mis-documentation. I however would like to flag this to the Github technical writers, for correction.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/ariesq">@AriESQ</a>. Welcome to the community!</p>
<p>If you search the curl documentation you link to for <code>--include</code>, you’ll notice that the short version for that option is <code>-i</code>.</p>
<p>I hope that helps!</p>

**Comment by AriESQ** [comment]
<p>Thank you, my mistake was that I searched for "-i " with a trailing space, and thus didn’t match with the man page entry <code>-i/--include</code></p>
